Src/FluidCaching/LifespanManager.cs

```diff
@@ -23,14 +25,21 @@ internal class LifespanManager<T> : IEnumerable<INode<T>> where T : class
         public LifespanManager(FluidCache<T> owner, int capacity, TimeSpan minAge, TimeSpan maxAge, GetNow getNow)
         {
             this.owner = owner;
-            double maxMS = Math.Min(maxAge.TotalMilliseconds, (double) 12 * 60 * 60 * 1000); // max = 12 hours
             this.minAge = minAge;
             this.getNow = getNow;
-            this.maxAge = TimeSpan.FromMilliseconds(maxMS);
-            validatyCheckInterval = TimeSpan.FromMilliseconds(maxMS / 240.0); // max timeslice = 3 min
-            bagItemLimit = Math.Max(capacity / 20, 1); // max 5% of capacity per bag
 
-            const int nrBags = 265; // based on 240 timeslices + 20 bags for ItemLimit + 5 bags empty buffer
+            // NOTE: Max = 12 hours
+            this.maxAge = TimeSpan.FromMilliseconds(Math.Min(maxAge.TotalMilliseconds, (double) 12 * 60 * 60 * 1000));
+
+            // NOTE: Max timeslice = 3 min
+            validatyCheckInterval =
+                TimeSpan.FromMilliseconds(Math.Min(maxAge.TotalMilliseconds, (double) 12 * 60 * 60 * 1000) / 240);
```

Comment:
I think we should be clear that this is `this.maxAge` divided by 240 instead of copying `Math.Min(maxAge.TotalMilliseconds, (double) 12 * 60 * 60 * 1000)`

Reply:
And we should define a constant for `240`